backend/Models/Response/SightingListResponse.cs

```diff
@@ -1,22 +1,27 @@
 using System.Runtime.InteropServices;
 using WhaleSpotting.Models.Data;
+using WhaleSpotting.Services;
 
 namespace WhaleSpotting.Models.Response;
 
 public class SightingListResponse()
 {
     public List<SightingResponse> Sightings { get; set; }
+    // private readonly IUserService _userService;
```

Comment:
Looks like this wasn't meant to be left in

Reply:
Deleted now.